koku/masu/openapi.json

```diff
@@ -847,10 +856,29 @@
           "type": "object",
           "properties": {
             "database": {
-              "type": "string"
+              "type": "string",
+              "nullable": "true"
```

Comment:
Again here I added this just to get the spec working with the returned data, I think we are returning an unexpected DB here and probably needs fixing. `"database_status": [
        {
            "database": null,
            "database_connections": 0
        },
        {
            "database": "template1",
            "database_connections": 0
        },
        {
            "database": "template0",
            "database_connections": 0
        },
        {
            "database": "postgres",
            "database_connections": 6
        },
        {
            "database": "hive",
            "database_connections": 0
        }`